packages/cadl-ranch-specs/http/azure/arm/models/common-types/managed-identity/identity.tsp

```diff
@@ -0,0 +1,141 @@
+  Expected query parameter: api-version=2023-12-01-preview
+
+  Expected response body:
+  ```json
+  {
+    "id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.Arm.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+    "location": "eastus",
+    "identity": {
+      "type": "SystemAssigned",
+      // this is for system-assigned
+      "principalId": <any uuid string>
+      // this is for system-assigned
+      "tenantId": <any uuid string>
+     },
+    "properties": {
+      "provisioningState": "Succeeded"
+    }
+  }
+  ```
+  """)
+  get is ArmResourceRead<ManagedIdentityTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+  Resource PUT operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.Arm.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  Expected query parameter: api-version=2023-12-01-preview
+  Expected request body:
+  ```json
+  {
+    "location": "eastus",
+    "identity": {
+      "type": "SystemAssigned"
+    },
+    "properties": {}
+  }
+  ```
+   Expected response body:
+  ```json
+  {
+    "id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.Arm.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+    "location": "eastus",
+    "identity": {
+      "type": "SystemAssigned",
+      // this is for system-assigned
+      "principalId": <any uuid string>,
+      // this is for system-assigned
+      "tenantId": <any uuid string>
+     },
+    "properties": {
+      "provisioningState": "Succeeded"
+    }
+  }
+  ```
+  """)
+  createWithSystemAssignedOnly is ArmResourceCreateOrReplaceAsync<ManagedIdentityTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+  Resource PATCH operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.Arm.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  Expected query parameter: api-version=2023-12-01-preview
+  Expected request body:
+  ```json
+  {
+    "identity": {
+      "type": "SystemAssigned,UserAssigned",
+      "userAssignedIdentities": {
+        "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1": {}
+      }
+    }
+  }
+  ```
+   Expected response body:
+  ```json
+  {
+    "id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.Arm.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+    "location": "eastus",
+    "identity": {
+      "type": "SystemAssigned,UserAssigned",
+      // this is user-assigned
+      "userAssignedIdentities": {
+        "<user-assigned-identity-id>": {
```

Comment:
This should be the same `"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1"`

Reply:
Fixed in the new version.